Automation/S3/delete_s3_buckets.py

```diff
@@ -0,0 +1,194 @@
+            elif delete_bucket_response == "Exit":
+                break
+
+            print()
+        
+        print()
+        if len(buckets_to_delete) > 0:
+            print("Attempting to delete buckets scheduled for deletion...")
+            delete_buckets(s3_client, buckets_to_delete)
+            print(Fore.GREEN + "Buckets deleted! Exiting script.")
+        else:
+            print(Fore.GREEN + "No buckets scheduled for deletion. Exciting script")
+        
+
+    except (ClientError, NoCredentialsError)  as e:
+        print(Fore.RED + f'Error connecting to S3! {e}')
+    
+
+def get_owner_email(s3_client, bucket_name):
+    '''
+    Gets the owner's email of an S3 bucket if that information is included in the bucket tags as OwnerEmail
+
+        Parameters:
+            s3_client (S3Client): the client necessary to execute S3 commands
+            bucket_name (str): the name of the bucket you would like the owner of
+        
+        Returns:
+            bucket_owner_email (str): Either the email of the owner of the bucket or No Owner if the no such email exists
+    '''
+    try:
+        get_bucket_tags_response = s3_client.get_bucket_tagging(
+            Bucket=bucket_name
+        )
+                    
+        bucket_owner_email_list = list(
+            filter(
+                lambda tag: tag['Key'] == "OwnerEmail",
+                get_bucket_tags_response['TagSet']
+            )
+        )
+
+        bucket_owner_email = "No Owner"
+        if len(bucket_owner_email_list) == 1:
+            bucket_owner_email = bucket_owner_email_list[0]['Value']
+
+        return bucket_owner_email
+
+    except s3_client.exceptions.from_code('NoSuchTagSetError') as e:
+        print(Fore.YELLOW + "Bucket does not have any tags!")
+
+def get_bucket_object_list(s3_client, bucket_name):
+    '''
+    Gets the names of the objects in an S3 bucket returns them as a list
+
+        Parameters:
+            s3_client (S3Client): the client necessary to execute S3 commands
+            bucket_name (str): the name of the bucket you would like the objects of
+        
+        Returns:
+            bucket_object_list (list): The list of object names
+    '''
+    try:
+        get_bucket_object_list_response = s3_client.list_objects_v2(
+            Bucket=bucket_name
+        )
+
+        bucket_object_list = []
+        if ('Contents' in get_bucket_object_list_response):
+            bucket_object_list = list(
+                map(
+                    lambda bucket_object: bucket_object['Key'],
+                    get_bucket_object_list_response['Contents']
+                )
+            )
+        else:
+            print(Fore.YELLOW + "Bucket is empty!")
+
+        return bucket_object_list
+
+    except s3_client.exceptions.from_code('NoSuchBucket') as e:
+        print(Fore.RED + "Bucket does not exist!")
+    
+
+def delete_buckets(s3_client, buckets_to_delete_list):
+    '''
+    Deletes the S3 buckets and their objects
```

Comment:
:) this is not very helpful comment - it is obvious from the name of the function. Maybe adding some details about the steps taken to delete the bucket will be helpful.